installer/assets/install_nightly_build

```diff
@@ -0,0 +1,145 @@
+#!/bin/sh
+# Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License"). You
+# may not use this file except in compliance with the License. A copy of
+# the License is located at
+#
+#     http://aws.amazon.com/apache2.0/
+#
+# or in the "license" file accompanying this file. This file is
+# distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF
+# ANY KIND, either express or implied. See the License for the specific
+# language governing permissions and limitations under the License.
```

Comment:
what is different about this file and https://github.com/aws/aws-sam-cli/blob/develop/installer/assets/install . If there are differences, can they be parameterized instead?

Reply:
This script is for customer to install the package. We don't want to introduce extra parameters to make the installation more complate. So I just copy that script and do necessary.

Reply:
The only differences are the default install path, and the binary name. The install script can replace them in place now.